bin/bibliotheca_monitor

```diff
@@ -6,5 +6,8 @@ bin_dir = os.path.split(__file__)[0]
 package_dir = os.path.join(bin_dir, "..")
 sys.path.append(os.path.abspath(package_dir))
 from core.scripts import RunCollectionMonitorScript
-from api.bibliotheca import BibliothecaEventMonitor
-RunCollectionMonitorScript(BibliothecaEventMonitor, cli_date=sys.argv[1:2]).run()
+from api.bibliotheca import BibliothecaEventMonitor, RunBibliothecaMonitorScript
+try:
+    RunBibliothecaMonitorScript(BibliothecaEventMonitor).run()
+except KeyboardInterrupt:
+    pass
```

Comment:
Why ignore this exception, especially since the script is going to end anyway?

Reply:
Mainly because it's annoying to someone running it on the command line, but I can take it out, so that we can see a traceback in case there's an infinite loop or similar.

Reply:
Yeah, I would take it out.

Reply:
Done.